docs/content/static/content/setup_notes

```diff
@@ -1,8 +1,6 @@
-This guide assumes that you have deployed Gloo to the `gloo-system` namespace and that the `glooctl` command line utility
-is installed on your machine. `glooctl` provides several convenient functions to view, manipulate, and debug Gloo resources;
-in particular, it is worth mentioning the following command, which we will use each time we need to retrieve the URL of
-the Gloo Gateway that is running inside your cluster:
+Before you begin, this guide assumes that you have the following setup.
```

Comment:
Reorganized into steps and added step 2 to enable discovery OR create your own upstreams as needed